Decred Wallet/Features/Wallets/Wallets.storyboard

```diff
@@ -401,12 +401,396 @@
+                                                <connections>
+                                                    <action selector="toggleAccountPropertiesDisplay:" destination="499-15-WzN" eventType="touchUpInside" id="i9w-Qh-Dta"/>
+                                                </connections>
+                                            </button>
+                                        </subviews>
+                                    </stackView>
+                                </subviews>
+                                <color key="backgroundColor" white="1" alpha="1" colorSpace="custom" customColorSpace="genericGamma22GrayColorSpace"/>
+                                <constraints>
+                                    <constraint firstItem="ViB-dv-2vV" firstAttribute="leading" secondItem="ggu-IF-Ehw" secondAttribute="leading" id="aMV-EI-kQE"/>
+                                    <constraint firstAttribute="trailing" secondItem="ViB-dv-2vV" secondAttribute="trailing" id="mVu-uR-o1A"/>
+                                    <constraint firstItem="ViB-dv-2vV" firstAttribute="top" secondItem="ggu-IF-Ehw" secondAttribute="top" id="uix-Xj-o3Q"/>
+                                    <constraint firstAttribute="bottom" secondItem="ViB-dv-2vV" secondAttribute="bottom" id="z8U-kM-Uhk"/>
+                                </constraints>
+                                <userDefinedRuntimeAttributes>
+                                    <userDefinedRuntimeAttribute type="boolean" keyPath="roundTopLeftCorner" value="YES"/>
+                                    <userDefinedRuntimeAttribute type="boolean" keyPath="roundTopRightCorner" value="YES"/>
+                                    <userDefinedRuntimeAttribute type="boolean" keyPath="roundBottomLeftCorner" value="NO"/>
+                                    <userDefinedRuntimeAttribute type="boolean" keyPath="roundBottomRightCorner" value="NO"/>
+                                    <userDefinedRuntimeAttribute type="number" keyPath="borderRadius">
+                                        <real key="value" value="14"/>
+                                    </userDefinedRuntimeAttribute>
+                                </userDefinedRuntimeAttributes>
+                            </view>
+                        </subviews>
+                        <color key="backgroundColor" white="0.0" alpha="0.0" colorSpace="custom" customColorSpace="genericGamma22GrayColorSpace"/>
+                        <constraints>
+                            <constraint firstItem="ggu-IF-Ehw" firstAttribute="leading" secondItem="ArT-Pb-cdS" secondAttribute="leading" id="1mp-JX-0HT"/>
+                            <constraint firstItem="ggu-IF-Ehw" firstAttribute="trailing" secondItem="ArT-Pb-cdS" secondAttribute="trailing" id="VKB-o2-RFR"/>
+                            <constraint firstItem="ArT-Pb-cdS" firstAttribute="bottom" secondItem="ggu-IF-Ehw" secondAttribute="bottom" id="mHX-Tv-9CH"/>
```

Comment:
The `secondItem` should be the `SafeArea` and the `StackView`'s `secondItem` should be the `superView`. See the suggestion below.
```suggestion
                            <constraint firstAttribute="bottom" secondItem="ViB-dv-2vV" secondAttribute="bottom" id="z8U-kM-Uhk"/>
```

Reply:
Funnily, that is actually relative to `Safe Area.bottom`. Not sure why that issue is there but will check.

Reply:
Ought to have been constraint to `Superview.bottom`. Fixed in https://github.com/raedahgroup/dcrios/pull/592/commits/8a5ceee3149eab96a16f24e38db46b4e80af8c65.